free-3.md

```diff
@@ -4,51 +4,96 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+A DSL to describe video game tactics. This would help newer plays learn from more
+experienced players by providing experienced players a way to share tactical or
+strategic advice.
+
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need met by this idea is the need to learn how to play games better. I'd be
+helping people who want share their strategies with others and those who want
+to learn.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL could be appropriate because it would provide a way for users to compare
+between different tactics and strategies.  In some games it could also help users
+understand losses by looking up the strategies that were used against them.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+This excites me because gaming is very fun and exciting and it's an activity that
+I love to do.  I think a number of Mudders also like games so it could be an idea
+that people here might think is fun.  Moreover there are many games that I am so
+clueless to the tactics, such as the game Diplomacy, that I can't really play well
+enought to enjoy.  If there was an easy way to learn or read about tactics it would
```

Comment:
In a lot of games, tactics (especially w.r.t things like the "best" tactic) can be rather subjective. I think that could complicate the language as well.